srcpkgs/lmdb++/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'lmdb++'
+pkgname=lmdb++
+version=0.9.14.0+20160229
+revision=1
```

Comment:
As `version` hasn't changed, this shouldn't be revived with an older revision. This has already been on `revision=2`:
`[-] lmdb++-0.9.14.0+20160229_2 C++11 wrapper for the LMDB database library (removed package)`

Reply:
Changed `revision=2`

Reply:
Should have bumped it to 3? :thinking:

Reply:
Yes

Reply:
done